app/Model/ObjectTemplate.php

```diff
@@ -316,4 +316,41 @@ public function setActive($id)
         }
         return 1;
     }
+
+    public function getRawFromDisk($uuidOrName)
+    {
+        $templates = $this->readTemplatesFromDisk();
```

Comment:
It seems pretty wasteful to read every single JSON objects just to find one of them.  Would you consider reading them and stopping when a match on UUID is found?

@Rafiot  what is the relationship of an object template directory name and the object template name?  Today the directory with the definitions.json file seems to set the name, and there's no way to have two directories of the same name below misp-objects/

Reply:
Alright, that should work fine then, as long as we're sure to never endup with two templates with the same name at any time in the future.

Reply:
Well the names *in* the files could overlap, but the names *of* the files never can.  So if the code checks for that case, it should be pretty solid.

Reply:
Maybe this is unnecessary optimization, but if the parameter is a name, the operation can be done in O(1) aka constant time - look for the file, read the file, return the contents (or error if no file).  If the parameter is a UUID, then the operation requires O(N) bcos you have to scan all the files.  I expect that the most frequent use case from the PyMISP side is to fetch an object template by name, because that's convenient to use in code, it's meaningful to people, etc.